### ISSUE TITLE
Add key combos like H for Help options to appear on screen

### ISSUE THREAD

**Issue by ahoneybun** [systemd/systemd#14438]
**Is your feature request related to a problem? Please describe.**

The problem is more about discovery about the shortcuts like + and - to add timeout.

**Describe the solution you'd like**

I'd like to have a few keys on the bottom to show these features or at least H for Help to show the other ones.

**Describe alternatives you've considered**

**Comment by poettering** [comment]
uh, what is this about?

Is this about sd-boot? If so, mention that!

sd-boot implements a "h" key that shows a help screen though already, which confuses me? What is this about hence?

Please elaborate?

**Comment by ahoneybun** [comment]
It is about sd-boot and I believe that the letter H should be displayed like this (H) Help on the screen to help with discovery of those other options.

**Comment by poettering** [comment]
I am not sure this is necessary. people technical enough to interact with a boot loader should sooner or later figure this out on their own, maybe just by trying.

**Comment by poettering** [comment]
(in general we prefer a clean look for this, and not show anything not really necessary.)

**Comment by ahoneybun** [comment]
There are people using Linux/BSD that aren't always technical and I didn't even think of pressing H until you mentioned it so I think it would be helpful. Feel free to close this or leave it open for more discussion if you would like.

**Comment by poettering** [comment]
Non-technical people really shouldn't interface with a boot loader directly, and technical people should be able to guess "h"

**Comment by ahoneybun** [comment]
They would if they are dual booting which is needed by some non-technical people. They do exist.

**Comment by thomas-zimmerman** [comment]
Than the UI for sd-boot is a failure. Even for technical people, code is not the documentation. The default for sd-boot does not have a timeout to show *any* UI, and when a user is pausing to see the menu, showing "Press (H) for help" is minimally helpful on the UI. Having the "help" text fully document how to use your software is stretch goal.

**Comment by poettering** [comment]
Well, I can live with you calling it a failure.

I mean, let's be real here. How many UIs do you know that prominently show "Press (H) for help"? Does firefox do that? does libreoffice do that? does inkscape do that?  To my knowledge there are pretty no real-life apps that do, but you insist that a thing showing a frickin menu needs to show that?

I mean, the help text doesn't really tell you much anyway besides either very obvious stuff or very technical stuff. But the technical stuff is fine to be hidden away under a key technical people would know and guess anyway and the trivial stuff is not really something people really need to be pointed to...

I mean, consider a UI like github, which is similarly focussed on technical people. If you press "?" there you get a help screen with keybindings. Do they tell you that you can do that, though? No not really, there#s no need really...

**Comment by ahoneybun** [comment]
You're listing applications that all have Help menus with documentation which prove the point that documentation in the application is beneficial to both technical users and non-technical users. At the end of the day we're all human/users and documentation is useful for every one of us.  

Displaying one line of text to help users discover useful features it not too much to ask for and I feel like it a requirement if you want more people to use it over another tool.

**Comment by thomas-zimmerman** [comment]
A self documenting UI is hard to get right but is user friendly. This bug is asking that we add a single line to document a feature that is already in the code-base and is not shown in a default install. The other major boot-loader (grub) will ask users to type help when in a emergency boot.
